### PR TITLE
JAMES-2545 Minor fixes for HealthChecks WebAdmin routes

### PR DESCRIPTION
**Pull request by chibenwa**
This follow the QA works started on James APIs

### CODE REVIEW

File: src/site/markdown/server/manage-webadmin.md [hunk 1]
```diff
@@ -78,16 +105,16 @@ Response codes:
...
 Will return the component's name, the component's escaped name, the health status and a cause.
 
 ```
 {
-  "componentName": "Cassandra Backend",
-  "escapedComponentName": "Cassandra%20Backend",
-  "status": "HEALTHY"
+  "componentName": "Cassandra backend",
+  "escapedComponentName": "Cassandra%20backend",
+  "status": "healthy"
```
Comment (blackheaven) [inline src/site/markdown/server/manage-webadmin.md]: I'm not really fan of changing interfaces

Comment (chibenwa) [reply src/site/markdown/server/manage-webadmin.md]: That's not changing interfaces that's changing inapropriate documentation!

Comment (blackheaven) [reply src/site/markdown/server/manage-webadmin.md]: so, it was a mistake in the documentation? curl requests will given the same result?

Comment (chibenwa) [reply src/site/markdown/server/manage-webadmin.md]: Bah yes of course!

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**